README_v1alphaX.md

```diff
@@ -28,7 +30,7 @@ The following environment variables need to be set for **Centos**:
 ```sh
 export IMAGE_CHECKSUM=http://172.22.0.1/images/centos-updated.qcow2.md5sum
 export IMAGE_NAME=centos-updated.qcow2
-export CAPI_VERSION=v1alpha2
+export CAPI_VERSION=v1alphaX
```

Comment:
it is a bit unclear that this one needs replacing while the script path should not be changed. maybe use something like "<v1alphaX>"

Reply:
rephrased